server/livecheck.py

```diff
@@ -118,8 +118,8 @@ def processCodeData(data):
     signature_file = makeTempFile(signature)
     signature_raw = run(['base64', '-d', signature_file], capture_output = True).stdout
     signature_raw_file = makeTempFile(signature_raw, "wb", None)
-
     verify_result = run(['openssl', 'dgst', '-sha256', '-verify', public_key_file, '-signature', signature_raw_file, message_file], capture_output=True)
+    cert_details = parseCertDetails(cert_file)
```

Comment:
I've moved the parsing below signature verification, but is it acceptable if we parse the cert here before we actually inspect the `verify_result` output below? 

I've currently set it up like this because it makes the file cleanup simpler than if we had parsed at line 138, since the parsing requires the temp `cert_file` to still exist

Reply:
Ah okay. Appreciate the further explanation of the issue. 

Updated the code [here](https://github.com/votingworks/livecheck-web/pull/2/commits/795a5ed60ee7d3e30cf85e65c8a1c228059641ec) – felt it was a clear way to ensure file cleanup happens properly with the different branches we have.